beetsplug/listenbrainz.py

```diff
@@ -199,17 +181,21 @@ def get_tracks_from_playlist(self, playlist):
         """This function returns a list of tracks in the playlist."""
         tracks = []
         for track in playlist.get("playlist").get("track"):
-            tracks.append(
-                {
-                    "artist": track.get("creator"),
-                    "identifier": track.get("identifier").split("/")[-1],
-                    "title": track.get("title"),
-                }
-            )
+            identifier = track.get("identifier")
+            if isinstance(identifier, list):
+                identifier = identifier[0]
+            artist = track.get(
+                "creator", "Unknown artist"
+            )  # Set a default value if 'creator' key is not present
+            track_dict = {
+                "artist": artist,
+                "identifier": identifier.split("/")[-1],
+                "title": track.get("title"),
+            }
+            tracks.append(track_dict)
```

Comment:
I find the original implementation slightly cleaner to read. Consider something like below
```suggestion
            identifier = track.get("identifier")
            if isinstance(identifier, list):
                identifier = identifier[0]

            tracks.append(
                {
                    "artist": track.get("creator", "Unknown artist"),
                    "identifier": identifier.split("/")[-1],
                    "title": track.get("title"),
                }
            )
```